runners/google-cloud-dataflow-java/build.gradle

```diff
@@ -86,3 +86,9 @@ dependencies {
 test {
   systemProperties = [ "beamUseDummyRunner" : "true" ]
 }
+
+// Generates :runners:google-cloud-dataflow-java:runQuickstartJavaDataflow
+def cloudProject = project.findProperty('cloudProject') ?: 'apache-beam-testing'
```

Comment:
cloudProject/cloudBucket is to generic since there are many clouds, how about gcpProject / gcsBucket

Reply:
Done. Changed it to gcpProject and gcsBucket/